package.json

```diff
@@ -9,51 +9,49 @@
     "react-redux"
   ],
   "author": "diana.suvorova@gmail.com",
-  "main": "dist/index.js",
+  "main": "index.js",
   "scripts": {
     "lint": "eslint ./lib ./tests",
-    "test": "npm run lint && mocha --compilers js:babel-register tests --recursive",
+    "test": "npm run lint && mocha tests --recursive",
     "semantic-release": "semantic-release",
-    "commitmsg": "npm run test && commitlint -e $GIT_PARAMS",
-    "build": "rm -fr ./dist && babel ./lib --out-dir ./dist --copy-files",
-    "prepublishOnly": "npm test && npm run build"
+    "commitmsg": "npm run test && commitlint -e $GIT_PARAMS"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/DianaSuvorova/eslint-plugin-react-redux"
   },
   "devDependencies": {
+    "@babel/core": "^7.17.0",
+    "@babel/eslint-parser": "^7.17.0",
     "@commitlint/cli": "^6.0.2",
     "@commitlint/config-conventional": "^6.0.2",
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.1.0",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
-    "eslint": "^4.14.0",
+    "eslint": "^8.8.0",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-config-standard": "^11.0.0-beta.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-node": "^5.2.1",
     "eslint-plugin-promise": "^3.6.0",
     "eslint-plugin-standard": "^3.0.1",
-    "husky": "^0.14.3",
-    "mocha": "^4.0.1",
+    "husky": "^7.0.4",
+    "mocha": "^9.2.0",
     "semantic-release": "^17.2.3"
   },
   "peerDependencies": {
-    "eslint-plugin-react": "^7.16.0"
+    "eslint-plugin-react": "^7.16.0",
```

Comment:
might as well hoist this up to the latest version of eslint-plugin-react also, since you're doing a breaking change anyways

Reply:
you don't have days off do you 😆

Reply:
every hour's an hour off from lots of things :-p there's just more than 24 hours worth of things to do in a day